internal/runbits/errors/errors.go

```diff
@@ -32,6 +33,19 @@ type OutputError struct {
 	error
 }
 
+var errorId string
+
+// Returns a relatively unique ID for error reporting.
+// We report this ID in non-user-facing errors and on Rollbar so we can cross-reference them.
+// Repeated calls return the same ID.
```

Comment:
This is fine because there can only ever be one error we report to the user that we'd have to lookup on Rollbar.